README.md

```diff
@@ -22,7 +22,8 @@ docker-machine create --driver amazonec2 kvstore
 ```
 eval $(docker-machine env kvstore)
 export KV_IP=$(docker-machine ip kvstore)
-docker run -d -p 8500:8500 -h consul --restart=always progrium/consul -server -bootstrap
+docker-machine ssh kvstore
```

Comment:
If you run `docker-machine ssh kvstore` and then `docker ps` you should see a similar output which means your containers are not running on your local machine. You could as well confirm that by running `eval $(docker-machine env default)` followed by a `docker ps` and you should not see a `pogrium/consul' container running.